tests/test_mailchimp_bookmarks.py

```diff
@@ -0,0 +1,212 @@
+import tap_tester.connections as connections
+import tap_tester.runner as runner
+from base import MailchimpBaseTest
+from tap_tester import menagerie
+
+
+class MailchimpBookMark(MailchimpBaseTest):
+    """Test that tap sets a bookmark and respects it for the next sync of a
+    stream."""
+
+    def name(self):
+        return "tap_tester_mailchimp_bookmark_test"
+
+    def test_run(self):
+        """Verify that for each stream you can do a sync that records
+        bookmarks.
+
+        That a second sync respects the bookmark     All data of the
+        second sync is >= the bookmark from the first sync     The
+        number of records in the 2nd sync is less than the first (This
+        assumes that         new data added to the stream is done at a
+        rate slow enough that you haven't         doubled the amount of
+        data from the start date to the first sync between         the
+        first sync and second sync run in this test) Verify that for the
+        full table stream, all data replicated in sync 1 is replicated
+        again in sync 2. PREREQUISITE For EACH stream that is
+        incrementally replicated, there are multiple rows of data with
+        different values for the replication key
+        """
```

Comment:
```suggestion
        """
        - Verify for each incremental stream you can do a sync which records bookmarks, and that the format matches expectations.
        - Verify that a bookmark doesn't exist for full table streams.
        - Verify the bookmark is the max value sent to the target for the a given replication key.
        - Verify 2nd sync respects the bookmark. All data of the 2nd sync is >= the bookmark from the first sync The number of records in the 2nd sync is less then the first
        """
```

Reply:
I have updated the expectations and assertions